vision/tests/system.py

```diff
@@ -617,263 +618,185 @@ def test_import_product_sets(self):
-        reference_image_id = "reference_image" + unique_resource_id()
-        reference_image_path = self.ps_client.reference_image_path(
-            project=PROJECT_OUTSIDE,
-            location=self.location,
-            product=product_id,
-            reference_image=reference_image_id,
-        )
-        self._verify_vpc_sc_error(
-            lambda: self.ps_client.get_reference_image(name=reference_image_path)
-        )
+        with pytest.raises(exceptions.PermissionDenied) as exc:
+            self.ps_client.get_reference_image(name=self.reference_image_path)
+
+        assert exc.value.message.startswith(_VPCSC_PROHIBITED_MESSAGE)
 
     def test_delete_reference_image_blocked(self):
-        product_id = "product" + unique_resource_id()
-        reference_image_id = "reference_image" + unique_resource_id()
-        reference_image_path = self.ps_client.reference_image_path(
-            project=PROJECT_OUTSIDE,
-            location=self.location,
-            product=product_id,
-            reference_image=reference_image_id,
-        )
-        self._verify_vpc_sc_error(
-            lambda: self.ps_client.delete_reference_image(name=reference_image_path)
-        )
+        with pytest.raises(exceptions.PermissionDenied) as exc:
+            self.ps_client.delete_reference_image(name=self.reference_image_path)
+
+        assert exc.value.message.startswith(_VPCSC_PROHIBITED_MESSAGE)
 
     def test_list_reference_images_blocked(self):
-        product_id = "product" + unique_resource_id()
-        product_path = self.ps_client.product_path(
-            project=PROJECT_OUTSIDE, location=self.location, product=product_id
-        )
-        self._verify_vpc_sc_error(
-            lambda: self.ps_client.list_reference_images(parent=product_path)
-        )
+        with pytest.raises(exceptions.PermissionDenied) as exc:
+            list(self.ps_client.list_reference_images(parent=self.product_path))
 
+        assert exc.value.message.startswith(_VPCSC_PROHIBITED_MESSAGE)
 
-@unittest.skipUnless(
-    BUCKET_OUTSIDE,
-    "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_BUCKET not set in environment.",
-)
+
+@vpcsc_config.skip_unless_inside_vpcsc
 class TestVisionClientVpcsc(VisionSystemTestBase):
     # Tests to verify Vision API methods are blocked by VPC SC when trying to access a gcs resource outside of a secure perimeter.
     def setUp(self):
-        VisionSystemTestBase.setUp(self)
+        super(TestVisionClientVpcsc, self).setUp()
         self.blocked_file = "LC08/01_$folder$"
         self.gcs_uri_blocked_file = "gs://{bucket}/{file}".format(
-            bucket=BUCKET_OUTSIDE, file=self.blocked_file
+            bucket=vpcsc_config.bucket_outside, file=self.blocked_file
         )
-        self._verify_vpc_sc_blocks_gcs_bucket()
         self.gcs_read_error_message = "Error opening file: gs://"
         self.gcs_write_error_message = "Error writing final output to: gs://"
```

Comment:
This vpc sc test is also broken, can you change this line to:

self.gcs_write_error_message = "Error writing final output to"